scipy-notebook/Dockerfile

```diff
@@ -40,13 +40,18 @@ RUN conda install --quiet --yes \
     'h5py=2.6*' \
     'vincent=0.4.*' \
     'beautifulsoup4=4.5.*' \
+    'protobuf=3.*' \
     'xlrd'  && \
     conda remove --quiet --yes --force qt pyqt && \
     conda clean -tipsy && \
     # Activate ipywidgets extension in the environment that runs the notebook server
     jupyter nbextension enable --py widgetsnbextension --sys-prefix && \
     fix-permissions $CONDA_DIR
 
+RUN git clone https://github.com/PAIR-code/facets.git
+WORKDIR facets
+RUN jupyter nbextension install facets-dist/ --user
```

Comment:
Guessing here, but I think what @parente meant is this should be `jupyter nbextension install --py facets-dist/ --sys-prefix`. Feel free to correct me, @parente.